apps/experiments/views/experiment.py

```diff
@@ -186,17 +186,17 @@ class Meta:
             "echo_transcript",
             "use_processor_bot_voice",
             "trace_provider",
-            "participant_whitelist",
+            "participant_allowlist",
         ]
-        labels = {"source_material": "Inline Source Material", "participant_whitelist": "Participant Whitelist"}
+        labels = {"source_material": "Inline Source Material", "participant_allowlist": "Participant Whitelist"}
```

Comment:
I think this is the part where we want to change Whitelist to Allowlist yeah?

```suggestion
        labels = {"source_material": "Inline Source Material", "participant_allowlist": "Participant Allowlist"}
```